Jenkinsfile.deploy

```diff
@@ -161,94 +161,101 @@ pipeline {
                     loginOpenshiftRegistry()
 
                     try {
-                        // Optaplanner taking a lot of resources, we should not build in parallel
-                        sh "make build-examples-images concurrent=1 ${getBDDParameters('never', false)}"
+                        sh "make build-examples-images tags='~@native' concurrent=3 ${getBDDParameters('never', false)}"
                     } catch(err) {
-                        unstable("Error building examples' images. Check the junit results.")
+                        unstable("Error building non-native examples' images. Check the junit results.")
                     }
-                }
-            }
-            post {
-                always {
-                    archiveArtifacts artifacts: 'test/logs/**/*.log', allowEmptyArchive: true
-                    junit testResults: 'test/logs/**/junit.xml', allowEmptyResults: true
-                }
-            }
-        }
-        stage('Run Full Testing') {
-            when {
-                expression { return shouldLaunchTests() }
-            }
-            steps {
-                script {
-                    // Catch and set unstable so the temp image is still pushed and we get the deployment properties, 
-                    // in case we decide to continue in the release
+                    
                     try {
-                        sh "make run-tests timeout=360 load_factor=3 concurrent=3 smoke=${params.SMOKE_TESTS_ONLY} ${getBDDParameters('always', true)}"
+                        // Optaplanner taking a lot of resources, we should not build in parallel
+                        sh "make build-examples-images tags='@native' concurrent=1 ${getBDDParameters('never', false)}"
```

Comment:
Won 10 minutes on build with concurrent=1 on native and 20 min with concurrent=2
but optaplanner native has been deactivated for now so not sure it will work with concurrent=2 once back ...

Reply:
nice :)